### DISCUSSION TITLE
Challenge in identifying S, I,R  for Vancomycin in Staphylococcus aureus isolates in my daatset

### DISCUSSION THREAD

**Discussion by sengeruan** [msberends/AMR#139] [Q&A]
Hi all,
I am doing data analysis using AMR Package Version 2.1.1. I have successfully executed the function as.mo() for bacterial identification, as.ab() for antibacterial identification and as.disk () for converting the diameters to disks in my dataset. However, when I executed as.sir() across the antibacterial columns with respective to bacterial names, only Staphylococcus aureus and CONS antibiotic susceptibility tests results returned "NA" instead of "S", or "I" or "R" as I need. Where did I went wrong? Anyone to help on this will be appreciated.

**Comment by msberends** [comment]
Hi, thanks for using this package! Can you provide a small example of how the data looks? At least the columns with the staphylococcal species and the antibiotics? Then I’m better able to help you. 

Many thanks!

**Comment by sengeruan** [comment]
Here is the file.
[staphy_vanco.xlsx](https://github.com/msberends/AMR/files/14748701/staphy_vanco.xlsx)

**Reply by msberends** [reply]
I also only get `NA`s:

```r
staph |> 
  mutate(VA = as.disk(VA),
         VA_SIR = as.sir(VA, ab = "vancomycin", mo = organism))
#> ℹ Run sir_interpretation_history() afterwards to retrieve a logbook with all the details of the breakpoint interpretations. Note that
#>   some microorganisms might not have breakpoints for each antimicrobial drug in EUCAST 2023.
#> 
#> Interpreting disk diffusion zones: 'vancomycin' (VAN), EUCAST 2023... OK 
#> # A data.frame: 15 × 4
#>      No_ organism                 VA VA_SIR
#>    <int> <chr>                 <dsk> <sir> 
#>  1     1 Staphylococcus aureus    17   NA  
#>  2     2 Staphylococcus aureus    30   NA  
#>  3     3 Staphylococcus aureus    16   NA  
#>  4     4 CNS                      NA   NA  
#>  5     5 Staphylococcus aureus    12   NA  
#>  6     6 Staphylococcus aureus     6   NA  
#>  7     7 Staphylococcus aureus    15   NA  
#>  8     8 Staphylococcus aureus    NA   NA  
#>  9     9 Staphylococcus aureus    22   NA  
#> 10    10 CNS                      18   NA  
#> 11    11 Staphylococcus aureus    18   NA  
#> 12    12 Staphylococcus aureus    NA   NA  
#> 13    13 CNS                      16   NA  
#> 14    14 CNS                       6   NA  
#> 15    15 Staphylococcus aureus    17   NA  ,
```

This is because vancomycin breakpoints are not available for disk diffusion zones. Not in EUCAST, and also not CLSI. You will have to use MIC to get interpretations.

**Comment by sengeruan** [comment]
Thanks Berends for your feedback.  So, it means breakpoints for some
microorganisms are not available, and alternatively we can use MIC values?
  Regards,
  Lameck Pashet.

   -----
  Email: ***@***.***
  Cellphone: +255 753 100 104 /  + 255 653 614 707
*   -----*

**Reply by msberends** [reply]
Exactly. There is not much of a choice...
